ert3/console/_console.py

```diff
@@ -106,6 +103,14 @@ def _get_ert3_examples_path():
     return pkg_examples_path
 
 
+def _get_ert3_example_names(pkg_examples_path):
```

Comment:
Could even stop taking an argument and just call `_get_ert3_examples_path` itself :)

Reply:
sure